tests/test_hook_services.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+
+from .basetest import HookTest
+
+
+class TestConfigureHookServices(HookTest):
+
+    def setUp(self):
+        super(TestConfigureHookServices, self).setUp()
+        
+        self.mock_systemctl = self.mock_binary("systemctl", """
+if [ "$1" = "is-enabled" ] && [ "$3" = "ssh.service" ]; then echo "disabled";exit 1; fi
+if [ "$1" = "is-active" ] && [ "$3" = "ssh.service" ]; then echo "inactive";exit 1; fi
+        """)
+
+    
+    def test_configure_service_disable(self):
+        self.mock_snapctl("service.ssh.disable", "true")
+        subprocess.check_call(["hooks/configure"])
+        self.assertEqual(self.mock_systemctl.calls(), [
+            ["systemctl", "--version"],
+            ["systemctl", "disable", "ssh.service"],
```

Comment:
Why didn't we go with the `disable --now` approach? I recall we had some issues in snapd but my memory is rusty.